arch/arm/src/armv7-a/arm_vectors.S

```diff
@@ -40,9 +40,19 @@
  ****************************************************************************/
 
 	.data
+
+	.globl	g_irqtmp
+
 g_irqtmp:
```

Comment:
can we avoid use the temporary storage? so SMP get resolved automatically.

Reply:
@xiaoxiang781216 

Hmm, I also tried to remove the temporary storage but I was not able to find how to do that.

Reply:
I have an idea that:

1. Switch to SVC mode directly in the entry point
2. Save some registers to stack
3. Switch back IRQ/FIQ/ABT/UND mode
4. Read LR/SPRS register
5. Switch to SVC mode again

Step 4 should has enough free registers saved at step 2.

Reply:
>I have an idea that:

@xiaoxiang781216 

Can you create a new PR for it?

Reply:
Ok, I will try this afternoon. We hit the same issue on dual Cortex-A7 SoC.